terraform/testcases/jaeger_mock/parameters.tfvars

```diff
@@ -1,6 +1,6 @@
 validation_config = "jaeger-mocked-server-validation.yml"
 
-sample_app_image = "aottestbed/aws-otel-jz-sample-app:v0.1.4"
+sample_app_image = "public.ecr.aws/aws-otel-test/jaeger-zipkin-sample-app:v0.1.5"
```

Comment:
Did you run the `jaegar_mock` test? I'm a bit surprised this works. With the mock tests, if the `sample_app_image` is provided, it'll try to pull the image. Usually, I've seen it complain about credentials because the mock module isn't set up to login to ECR.

Reply:
This is public ecr

Reply:
I did test on my mac but maybe I was still logged in when I ran this. I still think this will work since it is public ecr and you should not have to log in to pull.

Reply:
Maybe I'm reading this wrong, but from the documentation, it looks like you need to authenticate in order to pull.
https://docs.aws.amazon.com/AmazonECR/latest/public/docker-pull-ecr-image.html

![image](https://user-images.githubusercontent.com/84729962/149197466-a4780fbc-9881-44b5-900f-4059a4cbb34e.png)

Reply:
Unfortunately "public" ECR does still effectively require login to avoid pull limits. It's a reason we can't run integration tests against PR builds still - GHCR wouldn't have that problem. GHCR used to have an operations problem where images were all scoped to the organization but now they are scoped to repositories so perhaps we can give it a revisit, I notice we do seem to use it for python images already

https://github.com/orgs/aws-observability/packages